app/src/org/commcare/dalvik/activities/UpdateUIState.java

```diff
@@ -194,8 +188,8 @@ protected void updateProgressText(String msg) {
     }
 
     protected void updateProgressBar(int currentProgress, int max) {
-        progressBar.setProgress(currentProgress);
         progressBar.setMax(max);
+        progressBar.setProgress(currentProgress);
```

Comment:
The fix: switching the order of these set method calls. That was a fun one to track down!